src/cdk-experimental/menu/menu-item-trigger.ts

```diff
@@ -206,10 +214,10 @@ export class CdkMenuItemTrigger implements OnDestroy {
     // that means that the parent menu is a menu bar since we don't put the menu bar on the
     // stack
     const isParentMenuBar =
-      !menuStack.closeSubMenuOf(this._parentMenu) && menuStack.peek() !== this._parentMenu;
+      !menuStack?.closeSubMenuOf(this._parentMenu) && menuStack?.peek() !== this._parentMenu;
```

Comment:
Could you verify that your calls to methods on menuStack? will work in an expected way if menuStack is not defined?

Reply:
In the instance that there is no menustack there's nothing to do and the `?` prevents the methods from being called.